### DISCUSSION TITLE
Why replace khal with calcurse?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#412] [Q&A] [answered]
Hello,

[desktop-settings version `14.0.0`](https://github.com/manjaro-sway/desktop-settings/releases/tag/14.0.0) includes [fix: remove broken khal](https://github.com/manjaro-sway/desktop-settings/commit/08a7432eba5c8ab92837963e1b06b4f9118e407f).

What exactly is broken in khal?

Now definitions contain `set $calendar $term_float calcurse`. Should `calcurse` have been installed automatically with the update to desktop-settings version `14.0.0`?

**Comment by boredland** [comment, marked as answer]
Hi @heapifyman, thanks for asking!

"Broken" was more of a general impression. I never figured out their caldav sync etc. 

`calcurse` should install with 14.0.1-25

Main reason for the switch: ease of use. I used khal for quite some time, but never started liking the interface. Just compare the initial view:

![image](https://user-images.githubusercontent.com/4662748/211164828-bad44544-66cd-40ac-873b-a124f33e9adb.png)

To me this looks a lot more "starter friendly" and more versatile at the same time, while the configuration of khal alone was extremely tedious.

**Reply by boredland** [reply]
That being said: you can just revert to khal setting it to `set $calendar $term_float khal interactive` in a user config.

**Reply by heapifyman** [reply]
Thanks for explaining.

> you can just revert to khal setting it to set $calendar $term_float khal interactive in a user config

Yes, already did ;)

> I never figured out their caldav sync etc.

I think khal itself does not have caldav support but relies on vdirsyncer for that. You tell vdirsyncer the path to your local calendar directory (`~/.local/share/khal/...`) and to your remote calendar (https://my.nextcloud.org/...) plus credentials, and then it syncs remote to your local and vice versa. khal just continues to work with your local calendar directory.

Anyway, [calcurse-caldav](https://calcurse.org/files/calcurse-caldav.html) is labeled as alpha software. Do you have any experience with it? Does it work reliably?
